README.md

```diff
@@ -0,0 +1,29 @@
+## Requirements
+
+[Install Ruby on Rails 5.1](http://railsapps.github.io/installrubyonrails-mac.html)
+
+This project assumes the following tools and their respective versions are installed:
+* chruby: v0.3.9
+* ruby-install: v0.6.1
+* Ruby: v2.4.1
+* RubyGems: v2.6.12
+* Rails: v5.1
+* MySQL: v5.7.19
+
+
+
+## Getting Started
```

Comment:
I don't think we need this section, but I don't mind keeping it. Maybe also add this?
> Install dependencies.
`bundle install`